tests/operator/operator.go

```diff
@@ -915,6 +920,31 @@ var _ = Describe("[sig-operator]Operator", Serial, decorators.SigOperator, func(
 					return nil
 				}, 60*time.Second, 1*time.Second).Should(BeNil())
 
+				By(fmt.Sprintf("Verifying firmware UUID for vm %s", vmYaml.vmName))
+				Eventually(func() error {
```

Comment:
```suggestion
				Eventually(func(g Gomega) {
```

Change to this and make g.Expect in the func instead of returning errors.

Reply:
done, but note that we should probably remove assertions in this test in the next release since, the mutator webhook will create a new uuid, and the assert here will fail the test:
`Expect(vm.Spec.Template.Spec.Domain.Firmware).To(BeNil())`